### PR TITLE
Switching to next releases (1.0.6 and 5.0.6)

### PR DESCRIPTION
**Pull request by mathieucarbou**
__Updates:__

- apis - 1.0.6.beta
- passthrough-testing - 1.0.6.beta
- config - 10.0.6.beta
- core - 5.0.6-beta
- galvan - 1.0.6-beta

Before merging this PR, tc-platform __5.0.5.beta__ needs to be released.

Notes:

- Needed to adapt to changes from Terracotta-OSS/terracotta-platform#132
- Using new test entity to test the state of the monitoring tree
- monitoring service is now a builtin service
- management packages have been "normalized"

### CODE REVIEW

**Comment by mathieucarbou** [comment]
Blocked by: Terracotta-OSS/tc-passthrough-testing#14

**Comment by anthonydahanne** [comment]
test this please - latest releases are available

**Comment by anthonydahanne** [comment]
LGTM
@ljacomet @lorban anything to say before we merge ?

**Comment by lorban** [comment]
LGTM

File: management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java [hunk 1]
```diff
@@ -91,9 +89,7 @@
 
 public class ClusteringManagementServiceTest {
```
Comment (ljacomet) [inline management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java]: Want to have a chat about that test - but not related to the changes proposed in this PR.

Comment (mathieucarbou) [inline management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java]: @ljacomet : ok! let me know when you have time :-)

**Pull request merged**